tests/test_downloader_handlers.py

```diff
@@ -177,6 +177,13 @@ def render(self, request):
         return request.content.read()
 
 
+class LargeChunkedFileResource(resource.Resource):
+    def render(self, request):
+        for i in range(1024):
+            request.write(b"\x00" * 1024)
+        return b""
```

Comment:
yes and no, I would like to understand in what case it makes a difference. But it is not relevant to the test case so I am fine if you want to move forward and merge it.